SETUP.md

```diff
@@ -1,44 +1,80 @@
-# Create an app
```

Comment:
Hi @peculiaruc , please add the heading back for clarity. Or is there a reason why you'd prefer to remove it?